app/services/mentions/create_all.rb

```diff
@@ -10,32 +10,55 @@ def self.call(*args)
 
     def call
       # Only works for comments right now.
+      mentioned_users = users_mentioned_in_notifiable_text_that_are_existing_and_not_its_owners
+
+      delete_mentions_removed_from_notifiable_text(mentioned_users)
+      create_mentions_for(mentioned_users)
+    end
+
+    private
+
+    def users_mentioned_in_notifiable_text_that_are_existing_and_not_its_owners
+      mentioned_usernames = extract_usernames_from_mentions_in_text
+
+      collect_existing_users(mentioned_usernames).
+        then do |existing_mentioned_users|
+          reject_notifiable_owner existing_mentioned_users
```

Comment:
```suggestion
          reject_notifiable_owner(existing_mentioned_users)
```
Just to follow the convention in the rest of this file/codebase :)